rj_msgs/msg/RobotIntent.msg

```diff
@@ -6,6 +6,11 @@ uint8 SHOOT_MODE_CHIP = 1
 uint8 TRIGGER_MODE_STAND_DOWN = 0
 uint8 TRIGGER_MODE_IMMEDIATE = 1
 uint8 TRIGGER_MODE_ON_BREAK_BEAM = 2
+uint8 TRIGGER_MODE_AT_END = 3
+
+uint8 DRIBBLER_MODE_OFF = 0
+uint8 DRIBBLER_MODE_ON = 1
```

Comment:
can we add some documentation on what this stuff means? i assume "neutral" means let the planner decide. perhaps "Default" or "unspecified" would be better names? but either way, i'd like comments explaining.